src/main/java/by/exadel/internship/controller/InternshipController.java

```diff
@@ -34,4 +32,29 @@ public GuestInternshipDTO getInternshipById(@PathVariable UUID internshipId) {
         return internshipService.getById(internshipId);
 
     }
+
+    // I think we need to come up with another mapping
+    @GetMapping("/deletedInternships")
```

Comment:
1. The standard best practice for REST APIs is to have a hyphen, not camelcase or underscores.
You can read discussion about it: https://stackoverflow.com/questions/10302179/hyphen-underscore-or-camelcase-as-word-delimiter-in-uris
2. Now your url looks like `/internships/deletedInternships` `/users/deletedUsers`- it looks confused.
For interactions with deleted entities I see 2 ways:
a) specify them as additional resource - ` /deleted-internships`, `/deleted-internships/{id}`
b) declare some additional resourse like `trash` which will contains deleted entities
Ex. `trash/internships` , `trash/internships/{id}`, `trash/forms`.
I prefer more the second option. 
@SergeyKoval @viktarkolbik may be, you can suggest other options or make opinion about previous two?

Reply:
I would suggest something like that (full url):
/internships/historical
/internships/historical{id}
/forms/historical

Reply:
Done